### PR TITLE
Resolved the name being sent to multipart by trimming it 

### PR DESCRIPTION
**Pull request by dlucidone**

### CODE REVIEW

File: nativescript-background-http/background-http.ios.ts [hunk 1]
```diff
@@ -317,7 +317,13 @@ class MultiMultiPartForm {
 			if (!this.fields[i].filename) {
 				results += CRLF + CRLF + this.fields[i].value + CRLF;
 			} else {
-				results += '; filename="' + this.fields[i].filename + '"';
+				/*
+				 * Here the filename being sent to the multipart request is trimmed to the name only
+				 */
+				var sendFileName = this.fields[i].filename.split("/");
+                sendFileName = sendFileName[sendFileName.length - 1] || '';
```
Comment (Ablankzin) [inline nativescript-background-http/background-http.ios.ts]: I propose that you indent

File: nativescript-background-http/background-http.ios.ts [hunk 2]
```diff
@@ -317,7 +317,13 @@ class MultiMultiPartForm {
...
+                results += '; filename="' + sendFileName + '"';
```
Comment (Ablankzin) [inline nativescript-background-http/background-http.ios.ts]: I propose that you indent

**Comment by dlucidone** [comment]
@Ablankzin Made changes as suggested

**Comment by dlucidone** [comment]
@Ablankzin When can I expect this pull will merge?

**Comment by radeva** [comment]
Same problem is solved in PR #69 which is already merged.

**Pull request closed without merging**